pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -33,6 +33,7 @@ def _expected_command(self, name, store=None, verify_xss=True):
 
         expected_statement = [
             "DEFAULT_STORE={}".format(store),
+            "SAVED_SOURCE_DIR='{}/test_root/log{}'".format(REPO_DIR, shard_str),
```

Comment:
Are these bockchoy commands changes supposed to be here?  Seems like they are not related to the other changes?

Reply:
I mentioned this at the end of the PR description; it saves the HTML of the page at the time a bok-choy test fails so it can be inspected for debugging.  bok-choy has been able to do this for a while now, we just forgot to enable it in edx-platform.